data/strings/strings.txt

```diff
@@ -16463,7 +16465,7 @@
   [mobile_data_description]
     comment = NOTE to translators: please synchronize your translation with the English one.
     tags = android,ios
-    en = Mobile internet is required for map update notifications and for displaying detailed information about places and bookmarks.
+    en = Mobile internet is required for map update notifications, displaying some detailed information & uploading edits.
```

Comment:
Again do not "&".

As an aside, "displaying detailed information" is a privacy leak and should have a warning to click through.

Reply:
"displaying detailed information" is about opening external websites from the place information page. What kind of privacy leak do you see here?

Reply:
When someone is using offline map data, information about where they are and what POIs they are looking at stays on device, and is not available to any third party.   The OM download server sees "this IP address downloaded a map of Country X at this time", but that's pretty coarse.

What I'm concerned about is that finding a POI  and clicking "more info" somehow and having it load a page with that POI's identity in it discloses that some device at that IP address is interested.  Given other surveillance, that's likely to be useful for targeted ads.   This is often not a big deal, but when I use something like OM or OsmAnd I don't expect information about my browsing around to leave the device.    I am trying to suggest that users should be aware that such an information disclosure will happen before they take whatever action triggers it.   I realize that these comments are from a +3 standard deviation paranoid!

If it's just that you can click on URLs and they are opened, that's ok (because it's a URL and what else would clicking do), and then it's not necessary to warn the user that asking their browser to open URLs won't work well if they are offline -- that's then not about OM.

Reply:
For any POI from OSM, no server requests are done in Organic Maps. They were done in MapsMe (likely to update user reviews), and this message was left from MapsMe times. As I wrote, now users must tap links manually.

Requests can be done now when you select your bookmark with an HTML description, and it displays images or loads some other remote content (music, video). But that's the user's responsibility, he/she created or imported this content.

We're also thinking about loading images for Wikipedia articles in the future if users allowed it.

Reply:
That all sounds reasonable, with wikipedia image loading being opt in, default off.   I didn't mean to assert that things were bad, just that the help text was hard to understand and that therefore the reality was unclear.

Reply:
Will remove that bit as discussed above 👍